src/test/scala/repository/AnsweredQuestionLedgerRepositorySpec.scala

```diff
@@ -0,0 +1,15 @@
+package repository
+
+import java.util.Date
+
+import model.AnsweredQuestionInsert
+
+class AnsweredQuestionLedgerRepositorySpec extends DbTest {
```

Comment:
Flip the script here so that `DbTest` is actually a method that runs a single time and takes `AnsweredQuestionLedgerRepositorySpec` and its ilk as arguments so I can add them with no issue.